codeception.yml

```diff
@@ -17,12 +17,15 @@ settings:
 coverage:
     enabled: true
     remote: true
-    c3_url: 'http://zed.de.project.local'
+    c3_url: 'http://zed'
```

Comment:
This file is like a global configuration that the client is supposed to change himself. But maybe it can be set via an env var, or getting from the config_local.php. Needs further checking